apps/frontpage/components/docs/mdx/code-snippets/utils/get-metadata.ts

```diff
@@ -56,6 +56,7 @@ export const getMetadata = async ({ path, activeVersion }: Props) => {
           const [key, value] = match
             .split('=')
             .map((part) => part.replace(/"/g, ''));
+          // @ts-ignore
```

Comment:
Nit: Better to use `@ts-expect-error` (which will throw an error if it is no longer needed for some reason) and provide an explanation.